docs/devsguide/new_dev_guide.rst

```diff
@@ -209,5 +211,374 @@ text (Vim, TextEdit, Nano, etc.) and doesn’t invisibly add characters to the
 file (like Word). The only important part is to write the file in a manner that’s 
 considered reStructuredText (check out http://sphinx-doc.org/rest.html). Then, 
 Sphinx will do everything else under the hood as described `here 
-<http://pyne.io/devsguide/website.html>`__. Finally, commit these changes to 
-your forked version and submit a pull request.
+<http://pyne.io/devsguide/website.html>`__. Before you make the pull request, 
+make sure that you include a news file named after the purpose of your pr 
+following the template located in the news folder. Finally, commit these changes to 
+your forked version and submit a pull request. 
+
+---------------------
+Making a Pull Request
+---------------------
+The pull request should contain a description of the the pull request, any labels 
+(such as bug, docs, test), projects (such as Transition to PyMOAB), and a reference
+to the issue that led to the pr (which you can do by inserting '@issue_number' to 
+the description.
+
+The more specific your description, the better you employ the labels, and 
+project selections the easier it will be for other contributers to review your work.
+Adding a reference to the issue the pull request will allow people to see the issue 
+inspired it, and any conversation about the issue.
+
+.. _Dependencies:
+
+==================
+PyNE Dependencies:
+==================
+
+Something to note before you begin the installation process is the version of Python 
+that you are using. PyNE is currently supported from Python 2 through Python 3.6 
+(NOT 3.7, which is coming in future updates). A comprehensive Python package is Anaconda 
+which provides a free installation for MacOS, Windows, and Linux. Follow the instructions 
+for your specific platform `here <https://docs.anaconda.com/anaconda/install/>`_ . Installing 
+through Anaconda will result in many of the packages and dependencies being satisfied.
+
+-------
+Website
```

Comment:
Recommend putting the PyNE Library dependencies first.  We expect many more people will want to build the library without building the docs/website, so those dependencies should be more prominent.

Reply:
I didn't think of that, but that makes a lot of sense to me too. I'll include it in the next commit.